musicbot/bot.py

```diff
@@ -1564,13 +1573,20 @@ async def update_now_playing_status(self, set_offline: bool = False) -> None:
             return
 
         playing = sum(1 for p in self.players.values() if p.is_playing)
-        paused = sum(1 for p in self.players.values() if p.is_paused)
+        if self.config.status_include_paused:
+            paused = sum(1 for p in self.players.values() if p.is_paused)
+        else:
+            paused = 0
         total = len(self.players)
 
         def format_status_msg(player: Optional[MusicPlayer]) -> str:
+            if not self.config.status_include_paused:
+                p = sum(1 for p in self.players.values() if p.is_paused)
```

Comment:
Shouldn't this be `if not p.is_paused`?

Reply:
Not here.  This handles the case where counting paused players for dynamic status is disabled, but a custom status might still want to use the `{n_paused}` variable.   
So this independently counts the number of paused players so it can be displayed but wont impact the logic.

Reply:
Oh okay, I simply saw the above line `if not self.config` and thought wait this isn't right lol, thought maybe you copy pasted it by accident.